wolox-books/src/app/screens/auth/screens/book-list/book-list.component.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../../../../scss/colors';
+@import '../../../../../scss/fonts';
+
+$book-width: 220px;
+$book-height: 260px;
+
+.books {
+  margin: 100px 300px;
```

Comment:
If you use this margins, the books won't be centered on the screen. Remember what you did on the layout training 😄